charts/prometheus/values.yaml

```diff
@@ -1298,6 +1298,7 @@ serverFiles:
         static_configs:
           - targets:
             - localhost:9090
+        metrics_path: {{.Values.server.prefixURL}}/metrics
```

Comment:
Self reference not allowed.

Reply:
Please change to:

```
        metrics_path: /metrics
```

So the default is used for all users and you can still change your path in values.yaml.

Reply:
At that point no need to even add that, users with a prefixURL will simply have to reimplement the entire     `scrape_configs` array

Reply:
True. 
Adding it would maybe be a bit more verbose and could help to point the next user to the right direction if he wants to change it.
In the end i have no strong opinion about to add it or not.

Reply:
Hmm, I'm pretty unhappy with this solution then. The thing is, since there is a "prefixURL" value, the user of the Helm Chart would expect since to "just work", instead of having to rewrite the entire (!) scrape_config array.

Maybe one of these days I can come up with another solution.

Reply:
> Hmm, I'm pretty unhappy with this solution then. The thing is, since there is a "prefixURL" value, the user of the Helm Chart would expect since to "just work", instead of having to rewrite the entire (!) scrape_config array.
> 
> Maybe one of these days I can come up with another solution.

Hello,
I am testing this and I can definitivley confirm that I looked at the prefixURL and expected it to "just work". I had to search through the repo and found this open issue to confirm my suspicion that it does not.